### PR TITLE
JDK-8300751: [17u] Remove duplicate entry in javac.properties

### PR DESCRIPTION
**Pull request by shruacha1234**
Signed-off-by: Shruthi <Shruthi.Shruthi1@ibm.com>

OpenJDK bug : https://bugs.openjdk.org/browse/JDK-8300751

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8300751](https://bugs.openjdk.org/browse/JDK-8300751): [17u] Remove duplicate entry in javac.properties

### Reviewers
 * [Christoph Langer](https://openjdk.org/census#clanger) (@RealCLanger - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk17u-dev pull/1096/head:pull/1096` \
`$ git checkout pull/1096`

Update a local copy of the PR: \
`$ git checkout pull/1096` \
`$ git pull https://git.openjdk.org/jdk17u-dev pull/1096/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1096`

View PR using the GUI difftool: \
`$ git pr show -t 1096`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk17u-dev/pull/1096.diff">https://git.openjdk.org/jdk17u-dev/pull/1096.diff</a>

</details>

### CODE REVIEW

**Comment by shruacha1234** [comment]
@RealCLanger 
Please review this PR

**Review by RealCLanger — APPROVED** [review]

**Comment by RealCLanger** [comment]
In JDK Updates releases, you must not integrate a change before there's maintainer approval. Please read the [process documentation](https://wiki.openjdk.org/display/JDKUpdates/How+to+contribute+or+backport+a+fix). Since it looks like you don't have JBS access, I can tag the issue for you. However, please provide a short text as for the reasoning of the backports, risks involved and testing done. Thanks

**Comment by RealCLanger** [comment]
@shruacha1234 did you lose interest here and in the other two PRs?

**Comment by shruacha1234** [comment]
@RealCLanger These changes remove duplicate keys from the javac properties file. Tier1 testing shows no issues. The risk is low.

**Comment by GoeLin** [comment]
@shruacha1234, please add [17u] to the bug heading as I did in the JBS issue.

**Comment by shruacha1234** [comment]
> @shruacha1234, please add [17u] to the bug heading as I did in the JBS issue.

done

**Pull request closed without merging**
